corehq/apps/domain/forms.py

```diff
@@ -1776,10 +1776,12 @@ def save(self, commit=True):
 class ProBonoForm(forms.Form):
     contact_email = MultiCharField(label=gettext_lazy("Email To"), widget=forms.Select(choices=[]))
     organization = forms.CharField(label=gettext_lazy("Organization"))
-    project_overview = forms.CharField(widget=forms.Textarea, label="Project overview")
+    project_overview = forms.CharField(widget=forms.Textarea(attrs={"style": "resize:vertical"}),
```

Comment:
Nitpick: weird spacing, it looks like `label` is an attribute of the form. It's more common to put the params on their own line:
```
project_overview = forms.CharField(
    widget=forms.Textarea(attrs={"style": "resize:vertical"}), label="Project overview"
)
```
or to line up the kwargs:
```
project_overview = forms.CharField(widget=forms.Textarea(attrs={"style": "resize:vertical"}),
                                   label="Project overview")
```